book-service-rest/src/main/java/com/example/Library/exceptions/UserIsBlocked.java

```diff
@@ -0,0 +1,7 @@
+package com.example.Library.exceptions;
+
+public class UserIsBlocked extends RuntimeException {
```

Comment:
it would be good to return 403 (Forbidden) response code in case of this Exception